.github/workflows/build-and-publish-images.yml

```diff
@@ -15,71 +15,80 @@ jobs:
   compute-matrix:
     runs-on: ubuntu-latest
     outputs:
-      LATEST_LINUX_VER: ${{ steps.latest-values.outputs.LATEST_LINUX_VER }}
-      LATEST_CUDA_VER: ${{ steps.latest-values.outputs.LATEST_CUDA_VER }}
-      LATEST_PYTHON_VER: ${{ steps.latest-values.outputs.LATEST_PYTHON_VER }}
       MATRIX: ${{ steps.compute-matrix.outputs.MATRIX }}
+      LATEST_LINUX_VER: ${{ steps.compute-matrix.outputs.LATEST_LINUX_VER }}
+      LATEST_CUDA_VER: ${{ steps.compute-matrix.outputs.LATEST_CUDA_VER }}
+      LATEST_PYTHON_VER: ${{ steps.compute-matrix.outputs.LATEST_PYTHON_VER }}
```

Comment:
I think this change should be reverted. The `compute-matrix` step doesn't output these values.

Reply:
Yeah that was a typo, should be `latest-values` not `compute-matrix`.